src/plugins/service-worker/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+const buildPaths = require('mwp-cli/src/commands/buildCommands/config/paths');
+const path = require('path');
+
+/*
+ * The service worker plugin will register a root-level route for serving the
+ * service worker script file.
+ *
+ * Disk location of the script file is determined by builder CLI config import.
+ */
+export default function register(
+	server: Object,
+	options: ?Object,
+	next: () => void
+) {
+	/*
+	 * Route for service worker script at top-level path. Depends on `Inert`
+	 * `path` must match client `serviceWorker.register` call - MWP provides this
+	 * in the `<ServiceWorker>` component
+	 */
+	server.route({
+		method: 'GET',
+		path: '/asset-service-worker.{localeCode}.js',
```

Comment:
I'm realizing that I don't understand the caching behavior that this implements. Before this is used in production we need all devs to be aware of how this will affect caching in production. 

I did do a quick google search and its my impression that caching behavior is implementation specific.